app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class SubscriptionsController < ApplicationController
+
+    def index
+        render json: { message: 'Hashes of data will get converted to JSON' }
```

Comment:
Please return all subscriptions here. Create a serializer that only returns the email attributes. Go over to app/serializers and take a look at the volunteer serializer and follow the same pattern. This info will be useful for exporting emails to CSV or whatever format so newsletters can be sent and all of that